pkg/cmdlets/errors/errors.go

```diff
@@ -127,33 +127,37 @@ func ReportError(err error, cmd *captain.Command, an analytics.Dispatcher) {
 
 	_, hasMarshaller := err.(output.Marshaller)
 
+	cmdName := cmd.Name()
+	childCmd, findErr := cmd.Find(os.Args[1:])
+	if findErr != nil {
+		logging.Error("Could not find child command: %v", findErr)
+	}
+
+	var flagNames []string
+	for _, flag := range cmd.ActiveFlags() {
+		flagNames = append(flagNames, fmt.Sprintf("--%s", flag.Name))
+	}
+
+	label := []string{cmdName}
+	if childCmd != nil {
+		label = append(label, childCmd.UseFull())
+	}
+	label = append(label, flagNames...)
+
 	// Log error if this isn't a user input error
+	var action string
 	if !locale.IsInputError(err) {
 		multilog.Critical("Returning error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
+		action = anaConst.ActCommandError
 	} else {
-		cmdName := cmd.Name()
-		childCmd, err := cmd.Find(os.Args[1:])
-		if err != nil {
-			logging.Error("Could not find child command: %v", err)
-		}
-
-		var flagNames []string
-		for _, flag := range cmd.ActiveFlags() {
-			flagNames = append(flagNames, fmt.Sprintf("--%s", flag.Name))
-		}
-
-		trigger := []string{cmdName}
-		if childCmd != nil {
-			trigger = append(trigger, childCmd.UseFull())
-		}
-		trigger = append(trigger, flagNames...)
-
-		logging.Debug("Reporting input error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
-		an.Event(anaConst.CatDebug, anaConst.ActInputError, &dimensions.Values{
-			Trigger: p.StrP(strings.Join(trigger, " ")),
-		})
+		action = anaConst.ActCommandInputError
 	}
 
+	logging.Debug("Reporting error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
```

Comment:
```suggestion
	logging.Debug("Reporting error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
```

Realize you were just copying this over, but seems this was a debugging statement that probably shouldn't have been pushed in the first place. Line 150 already addresses logging non-input errors. And there's no real value in logging input errors to the log file imo.